condacolab.py

```diff
@@ -114,6 +115,9 @@ def install_from_url(
         For example, a value with spaces should be passed as::
 
             env={"VAR": '"a value with spaces"'}
+    pre_kernel_launch_script
```

Comment:
Let's call it `pre_conda` for simplicity
```suggestion
    pre_conda
```

Reply:
Apply change in the other instances as well.